src/vs/platform/backup/electron-main/backupMainService.ts

```diff
@@ -74,13 +74,17 @@ export class BackupMainService implements IBackupMainService {
 				const legacyBackupWorkspacesPath = join(this.backupHome, 'workspaces.json');
 				const legacyBackupWorkspaces = await Promises.readFile(legacyBackupWorkspacesPath, 'utf8');
 
-				await Promises.unlink(legacyBackupWorkspacesPath);
+				try {
+					await Promises.unlink(legacyBackupWorkspacesPath);
+				} catch (error) {
+					// ignore
```

Comment:
Should be more appropriate.
```suggestion
					console.error(error)
```